python/tvm/auto_scheduler/measure.py

```diff
@@ -303,12 +312,26 @@ class LocalBuilder(ProgramBuilder):
         This is used in a wrapper of the multiprocessing.Process.join().
     n_parallel : int = multiprocessing.cpu_count()
         Number of threads used to build in parallel.
-    build_func : str = 'default'
-        The name of registered build function.
+    build_func: callable or str = "default"
+        If is 'default', use default build function
+        If is 'ndk', use function for android ndk
+        If is callable, use it as custom build function, expect lib_format field.
     """
 
     def __init__(self, timeout=15, n_parallel=multiprocessing.cpu_count(), build_func="default"):
-        self.__init_handle_by_constructor__(_ffi_api.LocalBuilder, timeout, n_parallel, build_func)
+        if build_func == "default":
```

Comment:
So this is designed to use the class static member?
Why not just use a `self.build_func`.

Reply:
when auto_scheduler.local_builder.build is called, the context can't get LocalBuilder class instance

Reply:
He should keep the state information of `build_func` between functions, so just `self.build_func` can  not meet the requirement because we will create one class instance.